### PR TITLE
Update DG for Architecture section

### PR DESCRIPTION
**Pull request by qreoct**
Change diagrams for Architecture section to match StudyTracker.

For Week 10 v1.3 task.

### CODE REVIEW

File: docs/diagrams/ArchitectureSequenceDiagram.puml [hunk 1]
```diff
@@ -7,10 +7,10 @@ Participant ":Logic" as logic LOGIC_COLOR
...
 
-user -[USER_COLOR]> ui : "delete 1"
+user -[USER_COLOR]> ui : "delete n/Central Library"
 activate ui UI_COLOR
 
-ui -[UI_COLOR]> logic : execute("delete 1")
+ui -[UI_COLOR]> logic : execute("delete n/Central Library")
 activate logic LOGIC_COLOR
 
 logic -[LOGIC_COLOR]> model : deletePerson(p)
```
Comment (mslevis) [inline docs/diagrams/ArchitectureSequenceDiagram.puml:16]: Perhaps this can be changed to deleteStudySpot as well

**Review by mslevis — CHANGES_REQUESTED** [review]
Seems good! Just missed out one line.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -61,15 +61,16 @@ Each of the four main components (also shown in the diagram above),
...
+Other components interact with a given component through its interface rather than the concrete class (reason: to prevent outside component's being coupled to the implementation of a component), as illustrated in the (partial) class diagram below.
 
 <img src="images/ComponentManagers.png" width="300" />
 
 The sections below give more details of each component.
 
 ### UI component
 
-The **API** of this component is specified in [`Ui.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/ui/Ui.java)
+The **API** of this component is specified in [`Ui.java`](https://github.com/AY2122S1-CS2103T-T09-1/tp/tree/master/src/main/java/seedu/address/logic)
```
Comment (limdanqi) [inline docs/DeveloperGuide.md:73]: for this it should be ui/Ui.java instead of logic?

**Comment by mslevis** [comment]
Other than updating the PlantUML, the diagrams need to be exported before the changes are reflected on the actual documentation. Refer to [here](https://se-education.org/guides/tutorials/plantUml.html)

**Review by mslevis — APPROVED** [review]

**Pull request merged**